core/shared/src/main/scala/io/udash/properties/single/ReadableProperty.scala

```diff
@@ -25,7 +25,7 @@ trait ReadableProperty[A] {
   def listenOnce(valueListener: A => Any): Registration
 
   /** Returns listeners count. */
-  private[properties] def listenersCount(): Int
+  def listenersCount(): Int
```

Comment:
AbstractReadableProperty is missing override keyword for some methods - including this one. It's harder to find usages of it that way.